css/components/card.css

```diff
@@ -30,26 +32,10 @@
   border: 8px white solid;
 }
 
-.card__bookmark__button:hover,
-.card__bookmark__button:active,
-.card__bookmark__button:focus,
-.card__bookmark__button:focus-visible {
-  filter: hue-rotate(90deg) brightness(2);
-}
-
 .card__bookmark__button--active {
   filter: hue-rotate(90deg) brightness(2);
```

Comment:
what's this kind of magic? :D

Reply:
:)  Focus shows the "focus-frame" when you use the Tab-Key on your keyboard - to make the site better accessible for people with disabilities